.mvn/maven.config

```
@@ -1,2 +1,2 @@
 -Pconsume-incrementals
--Pmight-produce-incrementals
+-Pmight-produce-incrementals
```

Final newline police 👮 
```suggestion
-Pmight-produce-incrementals

```